src/pages/AboutPage.js

```diff
@@ -19,17 +19,16 @@ const AboutPage = () => (
         <h3 className="about-subheader logotype">Building pro bono projects to support Black lives</h3>
         <p>
           Build for Black Lives is a platform built to connect anti-racist individuals with activists, organizers, and
-          black-owned businesses to provide pro bono work creating websites, apps, graphics, social media content, data
-          visualizations, and/or other projects. Now more than ever, it has become essential for businesses and
-          organizations to have an online presence. Build for Black Lives aims to reduce discrimination against Black
-          folks when searching for tech and design services.
+          Black-owned businesses to provide pro bono work creating websites, apps, graphics, social media content, data
+          visualizations, and other projects. Our aim is to provide pro-bono services as a means of working towards
```

Comment:
we don't hyphenate pro bono in our tagline, maybe reconsider? can keep if wanted